clang/lib/Basic/SourceManager.cpp

```diff
@@ -527,17 +527,6 @@ FileID SourceManager::getNextFileID(FileID FID) const {
 
 /// Create a new FileID that represents the specified file
 /// being \#included from the specified IncludePosition.
-///
-/// This translates NULL into standard input.
-FileID SourceManager::createFileID(const FileEntry *SourceFile,
```

Comment:
Were there no callers that could pass `NULL`?

Reply:
Not that I could find.